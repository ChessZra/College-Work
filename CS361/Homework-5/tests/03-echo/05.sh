#!/bin/bash

PORT=$@

REQUEST=$'GET /echo HTTP/1.1\r
GET: /echo HTTP/1.1\r
Val: GET /echo HTTP/1.1\r\n\r\n'

printf "$REQUEST" | nc -N 127.0.0.1 $PORT
