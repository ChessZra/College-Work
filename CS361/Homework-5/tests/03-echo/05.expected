HTTP/1.1 200 OK
Content-Length: 44

GET: /echo HTTP/1.1
Val: GET /echo HTTP/1.1